ethereum/api/src/test/java/org/hyperledger/besu/ethereum/api/jsonrpc/JsonRpcHttpServiceTest.java

```diff
@@ -1367,6 +1367,19 @@ public void wrongJsonType() throws Exception {
     }
   }
 
+  @Test
+  public void emptyJson_shouldHandleNPE() throws Exception {
+    final RequestBody body = RequestBody.create(JSON, "");
```

Comment:
Is the original issue caused by an empty body? Does having no content-length in the request headers mean request body is empty?

Reply:
if I run the new test without the fix I get the same NPE
```
2022-01-19 06:14:59.416+10:00 | vert.x-eventloop-thread-0 | ERROR | RoutingContext | Unhandled exception in router
java.lang.NullPointerException: null
	at org.hyperledger.besu.ethereum.api.jsonrpc.JsonRpcHttpService.handleJsonRPCRequest(JsonRpcHttpService.java:572) ~[main/:?]
	at io.vertx.ext.web.impl.RouteState.handleContext(RouteState.java:1128) ~[vertx-web-4.2.1.jar:4.2.1]
	at io.vertx.ext.web.impl.RoutingContextImplBase.iterateNext(RoutingContextImplBase.java:117) ~[vertx-web-4.2.1.jar:4.2.1]
	at io.vertx.ext.web.impl.RoutingContextImpl.next(RoutingContextImpl.java:141) ~[vertx-web-4.2.1.jar:4.2.1]
	at org.hyperledger.besu.ethereum.api.handlers.TimeoutHandler.processHandler(TimeoutHandler.java:75) ~[main/:?]
```

Reply:
tested it out with postman and it does fix the problem.